docs/user-guide/update-an-existing-project.md

```diff
@@ -29,6 +29,30 @@ curl -sSL https://raw.githubusercontent.com/frequenz-floss/frequenz-repo-config-
 Make sure that the version (`{{ ref_name }}`) matches the
 target version you are migrating to.
 
+!!! Note
```

Comment:
Optional: Maybe making it a *tip* is more suitable. You can also add a custom title, for example:

```suggestion
!!! Tip "Tip for MacOS users"
```